libpromises/evalfunction.c

```diff
@@ -8136,6 +8142,41 @@ void ModuleProtocol(EvalContext *ctx, char *command, const char *line, int print
         }
         break;
 
+    case '&':
+        // TODO: the variable name is limited to 256 to accommodate the
+        // context name once it's in the vartable.  Maybe this can be relaxed.
+        sscanf(line + 1, "%256[^=]=%4095[^\n]", name, content);
+
+        if (CheckID(name))
+        {
+
+            if (FileCanOpen(content, "r"))
+            {
+                int size_max = IntFromString("inf");
```

Comment:
This can be `const`.

Reply:
I just copied this from another place. I can adjust it here